### DISCUSSION TITLE
cnCouchbaseUrl value not using override value

### DISCUSSION THREAD

**Discussion by ksully-execute** [JanssenProject/jans#6885] [Q&A] [answered]
fresh cluster (on-prem), with override file including Couchbase coniguration (used before successfully), specifies these values
`  configmap:
     cnCouchbaseBucketPrefix: jans
     cnCouchbaseCrt:  <removed>
     cnCouchbaseIndexNumReplica: 0
     cnCouchbasePassword: <somepass>
     cnCouchbaseSuperUser: <someuser>
     cnCouchbaseSuperUserPassword: <somepass>
     cnCouchbaseUrl:  couchbasedev-0000.couchbase.svc.cluster.local
     cnCouchbaseUser: <someuser`
     But when started, the cluster errors out as it can't find the couchbase instance, but it's using the wrong value.
The value shown below is present in the default chart values file (which I used previously to get the thing up but found the same issue when I replaced the couchbase definition, 
`    cnCouchbaseUrl: cbjanssen.default.svc.cluster.local`
Have cleaned up all k8s reosurces, there's nothing that has that 'cbjanssen' defined, and the override is spelled right, etc. and is coming in on the helm command correctly.

So why is that value not taking, any thoughts?

**Comment by mzico** [comment, marked as answer]
https://github.com/JanssenProject/jans/discussions/10222
